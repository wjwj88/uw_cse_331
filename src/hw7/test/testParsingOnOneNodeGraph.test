# Tests building a graph with only one node and no edges hw7

# Load oneNodeGraph.tsv
LoadGraph oneNodeGraph oneNodeGraph.tsv

# Print nodes in the graph and edges from the node
ListNodes oneNodeGraph
ListChildren oneNodeGraph Amanda