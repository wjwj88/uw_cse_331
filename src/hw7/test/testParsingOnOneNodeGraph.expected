# Tests building a graph with only one node and no edges hw7

# Load oneNodeGraph.tsv
loaded graph oneNodeGraph  

# Print nodes in the graph and edges from the node
oneNodeGraph contains: Amanda
the children of Amanda in oneNodeGraph are: